src/components/PokemonInfoCard.tsx

```diff
@@ -0,0 +1,127 @@
+            <Container className="mt-2 mr-5">
+                <Row>
+                    <PokemonCard
+                        pokemonInfo={pokemonInfo}
+                        isLoading={isLoading}
+                    />
+                    <Col className="mx-lg-3">
+                        <h3 className="mt-4">Abilities:</h3>
+                        
+                        {pokemonInfo.abilities.map((ability, index) => {
+                            return (
+                                <Accordion key={ability.name} defaultActiveKey={ability.name}>
+                                    <Accordion.Header>
+                                        {ability.name}
+                                    </Accordion.Header>
+                                    {ability.definition && (<Accordion.Body>{ability.definition}</Accordion.Body>)}
+                                </Accordion>
+                        )})}
+
+                        <h3 className="mt-4">Stats:</h3>
+                        <div className="mx-lg-3 mx-sm-2">
+                            <h4 className="fs-5">HP:</h4>
+                            <div className="mx-2 ">{pokemonInfo.hp}</div>
+                            <h4 className="fs-5">Attack:</h4>
+                            <div className="mx-2 ">{pokemonInfo.attack}</div>
+                            <h4 className="fs-5">Special Attack:</h4>
+                            <div className="mx-2 ">{pokemonInfo.specialAttack}</div>
+                            <h4 className="fs-5">Defense:</h4>
+                            <div className="mx-2 ">{pokemonInfo.defense}</div>
+                            <h4 className="fs-5">Special Defense:</h4>
+                            <div className="mx-2 ">{pokemonInfo.specialDefense}</div>
+                        </div>
+                    </Col>
+                    <Col>
+                        <h3 className="mt-4">Games Appeared In:</h3>
+                        <ul>
+                        {pokemonInfo.gamesIn.map((game, index) => (
+                            <li key={index}>{game}</li>
+                        ))}  
+                        </ul>
+                    </Col>
+                </Row>
+                <Row>
+                    <h2 className="text-center mt-5">View other Pokemon:</h2>
+                    <PokemonSearch
+                        numOfPokemon={3}
+                    />
+                </Row>
+            </Container>
+            ): (<LoadingSpinner/>)}
+            
+        </div>
+    )
+
+};
+
+const PokemonCard = ({
+    pokemonInfo,
+    isLoading
+}: Props) => {
+    return (
+        <Card className="shadow-sm">
+            <Card.Body>
+                <Row>
+                    <Col>
+                        <Card.Title className="fs-2">{pokemonInfo?.name} - {pokemonInfo?.id}</Card.Title>
+                        <Image
+                            src={pokemonInfo?.sprite ?? "src/data/pikachu.jpg"}
```

Comment:
for Next JS, the image will have to be in the `public` folder on built, so let's move the image there and use the relative path from `public`